sdk/php/template/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "daggermodule/example",
+  "description": "An example module you can use as a starting point for building your own dagger modules",
+  "minimum-stability": "dev",
+  "license": "proprietary",
+  "authors": [
+  ],
+  "repositories": [
+    {
+      "type": "path",
+      "url": "./sdk"
+    }
+  ],
+  "require": {
+    "php": "^8.1",
+    "dagger/dagger": "*@dev"
```

Comment:
I believe we intended to swap that to a stable version once we'd got a working SDK but had become forgotten tech-debt.

Resolved and based on "^0.12.0", I will see about basing it off of "^0.12.2" after solving other issues.

Reply:
This has to be `*@dev` because it's installing the package from a directory inside the Container. Because it's installing it from a directory, it doesn't consider it a "tagged" version, so we can't use a tag.